l10n_cl_electronic_invoicing/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,151 @@
+.. |step1| image:: ./static/images/01-res.company.png
+    :alt: Company form view
+    :width: 400px
+
+.. |step2| image:: ./static/images/02-res.company.png
+    :alt: Company form view
+    :width: 400px
+
+.. |step3| image:: ./static/images/03-res.company.png
+    :alt: Company form view
+    :width: 400px
+
+.. |step4| image:: ./static/images/04-res.company.png
+    :alt: Company form view
+    :width: 400px
+
+.. |step5| image:: ./static/images/05-account.journal.png
+    :alt: Accounting Journal
+    :width: 400px
+
+.. |step6| image:: ./static/images/06-account.journal.png
+    :alt: Accounting Journal
+    :width: 400px
+
+.. |step7| image:: ./static/images/07-account.journal.png
+    :alt: Accounting Journal
+    :width: 400px
+
+.. |step8| image:: ./static/images/08-ir.sequence.png
+    :alt: Sequence form view
+    :width: 400px
+
+.. |step9| image:: ./static/images/09-ir.sequence.png
+    :alt: Sequence form view
+    :width: 400px
+
+.. |step10| image:: ./static/images/10-fetchmail.server.png
+    :alt: Fetchmail Server
+    :width: 400px
+
+.. |step11| image:: ./static/images/11-res.config.settings.png
+    :alt: Accounting Settings
+    :width: 400px
+
+.. |step12| image:: ./static/images/12-res.company.png
+    :alt: Company form view
+    :width: 400px
+
+.. |step13| image:: ./static/images/13-www.sii.cl.png
+    :alt: SII form
+    :width: 400px
+
+Please note that the configuration for the production environment is different
+than for a test one. The difference resides in the following fields:
+
+* Resolution Number
+* Resolution Date
+* DTE Service Provider
+* Folios
+
+Companies
+---------
+
+To configure this module, you need to:
+
+* Go to Settings > Users & Companies > Companies
+* Select or create a company
+* Enter all the information of the company especially:
+
+  * the logo
+  * the resolution number
+  * the resolution date
+
+|step1|
+
+* Go to the Signature File tab
+* Upload your digital certificate
+
+|step2|
+
+* Edit to update the Subject Serial Number with your RUT
+
+|step3|
+
+* Select users allowed to send and receive invoices
+
+|step4|
+
+Journals
+--------
+
+* Go to Accounting > Configuration > Journals
+* Select the Customer Invoices or Vendor Bills journal
+* Enter the documents that can be generated
+
+|step5|
+
```

Comment:
@max3903 Step 5 is empty. :thinking:

Reply:
@bodedra What do you mean by empty? The image is declared with the others. Once the README.rst will be generated at the root of the repo, image paths should work.

Reply:
ah. I got it. I was expected content like have in others.

My comment is invalid. :smile: